### DISCUSSION TITLE
New files with GD ?

### DISCUSSION THREAD

**Discussion by artineo** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2425] [General]
What is difference "BIGTREE_GD_TFT35_V3.0_E3.27.x.bin" and "BIGTREE_TFT35_V3.0_E3.27.x.bin" (without GD) ?

**Comment by 3DSmitty** [comment]
I am trying to figure out the same thing. I have a TFT35 V2.0 and there is only a GD bin for it. What is the GD???

**Comment by guruathwal** [comment]
**BIGTREE_GD_TFT35_V3.0_E3.27.x.bin** is for a different TFT35 V2 variant with a GD32 processor. Use **BIGTREE_TFT35_V3.0_E3.27.x.bin** for TFT35 V2 with STM32 processor.
